packages/block-editor/src/store/private-selectors.js

```diff
@@ -63,10 +63,7 @@ export const isBlockSubtreeDisabled = createSelector(
 				)
 			);
 		};
-		return (
-			getBlockEditingMode( state, clientId ) === 'disabled' &&
-			getBlockOrder( state, clientId ).every( isChildSubtreeDisabled )
-		);
+		return getBlockOrder( state, clientId ).every( isChildSubtreeDisabled );
```

Comment:
Note that this is a private selector, so we can change it. There's also only one used case which is the above.